lib/tarball.rb

```diff
@@ -25,7 +25,7 @@ def list
     output = LoggedCheetah.run("tar", "tvf", @file, :stdout => :capture)
 
     output.lines.map do |line|
-      mode, user_and_group, size, date, time, *rest = line.split(" ")
+      mode, user_and_group, size, date, time, *rest = line.split(" ", 6)
```

Comment:
Useless assignment to variable - `date`. Use `_` or `_date` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `time`. Use `_` or `_time` as a variable name to indicate that it won't be used.